analyzer/quality-analyzer/src/main/java/eu/fasten/analyzer/qualityanalyzer/QualityAnalyzerPlugin.java

```diff
@@ -202,8 +186,7 @@ public void setPluginError(Exception throwable) {
 
         @Override
         public long getMaxConsumeTimeout() {
```

Comment:
Why is the consume timeout decreased? I assume QualityAnalyzer is faster now and doesn't need a 1h timeout value.